README.md

```diff
@@ -6,44 +6,50 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
+* Install Fluentd
+  * [Download here](http://www.fluentd.org/download)
+  * If using fluentd package manager (td-agent): `td-agent-gem install fluent-plugin-logdna`
+  * To install without td-agent: `gem install fluent-plugin-logdna`
+* Add the config below to `/etc/fluent/fluent.conf`. For td-agent, `/etc/td-agent/td-agent.conf`:
+
+## Configuration
+
+### Configuration Parameters
+- `api_key`: [Ingestion Key](https://docs.logdna.com/docs/ingestion-key) *Required*
+- `hostname`: Hostname *Required*
+- `app`: App Name *Optional*
+- `mac`: MAC Address *Optional*
+- `ip`: IP Address *Optional*
+- `tags`: Comma-Separated List of Tags *Optional*
+- `request_timeout`: Request Timeout *Optional*
+  - **Note**: Supports `s` and `ms` Suffices
+  - **Default**: `30 s`
+- `ingester_domain`: Custom Ingester URL *Optional*
+  - **Default**: `htttps://logs.logdna.com`
+
+### Sample Configuration
 
 ~~~~~configuration
 <match **>
   @type logdna
-  api_key xxxxxxxxxxxxxxxxxxxxxxxxxxx        # paste your api key here (required)
-  hostname "#{Socket.gethostname}"           # your hostname (required)
-  app my_app                                 # replace with your app name
-  #mac C0:FF:EE:C0:FF:EE                     # optional mac address
-  #ip 127.0.0.1                              # optional ip address
-  #tags web,dev                              # optional tags
-  request_timeout 30000 ms                   # optional timeout for upload request, supports seconds (s, default) and milliseconds (ms) suffixes, default 30 seconds
-  buffer_chunk_limit 1m                      # do not increase past 8m (8MB) or your logs will be rejected by our server.
-  flush_at_shutdown true                     # only needed with file buffer
+  api_key xxxxxxxxxxxxxxxxxxxxxxxxxxx
+  hostname "#{Socket.gethostname}"
+  app my_app
+  mac C0:FF:EE:C0:FF:EE
+  ip 127.0.0.1
+  tags web,dev
+  request_timeout 30000 ms
+  ingester_domain https://logs.logdna.com
 </match>
 ~~~~~
 
 * Restart fluentd to pick up the configuration changes.
 * `sudo /etc/init.d/td-agent stop`
 * `sudo /etc/init.d/td-agent start`
 
-### Recommended Configuration Parameters
-
-* buffer_type
-  * We recommend setting this to memory for development and file for production (file setting requires a buffer_path).
-* buffer_queue_limit, buffer_chunk_limit
-  * We do not recommend increasing buffer_chunk_limit past 8MB.
-* flush_interval
-  * Default is 60s. We recommend keeping this well above 5s.
-* retry_wait, max_retry_wait, retry_limit, disable_retry_limit
-  * We recommend increasing these values if you are encountering problems.
-
 ### Options
 
-* App name and log level can also be provided on a line-by-line basis over JSON:
+* App name and log level can be specified per each line via JSON
 * `_app` and `level` will override the config
 
 If you don't have a LogDNA account, you can create one on [https://logdna.com](https://logdna.com) or if you're on macOS w/[Homebrew](https://brew.sh) installed:
```

Comment:
why do we mention macOS w/Homebrew? Doesn't feel relevant

Reply:
@alfeng6 I asked the same thing. Ignored?

Reply:
I'm removing it.

Reply:
@johnpadilla, wait 😄  yea, going to remove the whole registration part - I don't find that part helpful since in order to `Ingestion Key` which is **Required**, you should have account.